apps/braze/src/locations/ConfigScreen.tsx

```diff
@@ -136,9 +136,47 @@ const ConfigScreen = () => {
             )}
           </Form>
         </Box>
+        <Subheading className={styles.subheading}>Add Braze to your content types</Subheading>
+        <InformationSection
+          url={CONTENT_TYPE_DOCUMENTATION}
+          linkText="here"
+          marginTop="spacing2Xs"
+          dataTestId="content-type-docs-here-link">
+          Navigate to the content type you would like to use under the Content model tab in the main
+          navigation. Select the content type and adjust the sidebar settings on the Sidebar tab.
+          Learn more about configuring your content type
+        </InformationSection>
       </Box>
     </Flex>
   );
 };
 
+type InformationSectionProps = {
+  url: string;
+  children: string;
+  linkText: string;
+  marginTop?: Spacing;
+  marginBottom?: Spacing;
+  dataTestId?: string;
+};
+function InformationSection(props: InformationSectionProps) {
+  return (
+    <Paragraph
+      marginBottom={props.marginBottom ? props.marginBottom : 'spacing2Xs'}
+      marginTop={props.marginTop ? props.marginTop : 'spacingXs'}>
+      {`${props.children} `}
```

Comment:
If the children is React.ReactNode, then you could:
```suggestion
      {props.children}
```